src/wallet/wallet.cpp

```diff
@@ -3432,11 +3432,11 @@ std::vector<CompactTallyItem> CWallet::SelectCoinsGroupedByAddresses(bool fSkipD
     }
 
     // debug
-    if (LogAcceptCategory(BCLog::SELECTCOINS, BCLog::Level::Debug)) {
+    if (LogAcceptDebug(BCLog::SELECTCOINS)) {
```

Comment:
this one is bitcoin's original code https://github.com/bitcoin/bitcoin/blob/master/src/wallet/coinselection.cpp#L736

Reply:
That's a different function introduced in https://github.com/dashpay/dash/pull/906